stdlib/json.gr

```diff
@@ -1342,7 +1353,9 @@ let rec readCodePoint = (bytePosition: Number, string: String) => {
 
 let codePointUTF8ByteCount = (usv: Number) => {
   if (!Char.isValid(usv)) {
-    throw InvalidArgument("Invalid unicode scalar value")
+    throw InvalidArgument(
+      "Impossible: JSON parser encountered an invalid unicode scalar value"
```

Comment:
Why is this impossible?

Reply:
This would be impossible in proper use of the parser as it is called in the `next` function which is working on a buffer a string. and is built in the `parse` function. So you would need a corrupt grain string in order to trigger it. which I think we consider impossible behaviour.